rust/src/types.rs

```diff
@@ -938,6 +938,128 @@ impl Type {
         let mut name = QualifiedName::from(name);
         unsafe { BnStr::from_raw(BNGenerateAutoDemangledTypeId(&mut name.0)) }
     }
+
+    pub fn demangle_gnu3<Mangled: BnStrCompatible>(
+        arch: &CoreArchitecture,
+        mangled_name: Mangled,
+        simplify: bool,
+    ) -> Result<(Option<Type>, Vec<String>)> {
```

Comment:
The Python version of this API gets it wrong; the return value should instead match the C++ version: `Result<(Option<Type>, QualifiedName)>`